betterproto/tests/test_inputs.py

```diff
@@ -50,14 +56,17 @@ def apply_xfail_marks(test_set: Set[str], xfail: Set[str]):
 test_cases = TestCases(
     path=inputs_path,
     services=test_input_config.services,
-    xfail=test_input_config.tests,
+    xfail=test_input_config.xfail,
 )
 
 plugin_output_package = "betterproto.tests.output_betterproto"
 reference_output_package = "betterproto.tests.output_reference"
 
+TestData = namedtuple("TestData", ["plugin_module", "reference_module", "json_data"])
```

Comment:
Python trivia of the day: [tuples are generally better than lists where mutability isn't needed](https://stackoverflow.com/questions/68630/are-tuples-more-efficient-than-lists-in-python)

So we can save some microseconds here with:
```
TestData = namedtuple("TestData", ("plugin_module", "reference_module", "json_data"))
```

Reply:
hm. i don't see the practical benefit to using tuples here. i find the list more readable as `()` are mostly used for function calls and it takes more staring to figure out `(   (  ) )` than `(   [  ] )`

for performance heavy tasks i'll agree with you though, and start using tuples whenever possible.

Reply:
Not much practical benefit, it's a bit of an academic point. I think it's good habit to default to tuples in general, though this doesn't seem to be a widespread opinion, and I buy that there's some readability trade off. No big deal.